tests/integration/m3-debug-adapter-test.js

```diff
@@ -241,51 +244,28 @@ module('integration/debug-adapter', function(hooks) {
     assert.equal(get(records, 'length'), 1, 'Correct number of models is returned');
   });
 
-  test('It handles adding new types dynamically', async function(assert) {
-    this.debugAdapter.addedType = this.sinon.stub();
-
-    this.owner.lookup('service:m3-schema').watchModelTypes = true;
-
-    this.store.pushPayload('com.example.newModel', {
-      data: {
-        id: 'urn:model:1',
-        type: 'com.example.newModel',
-        attributes: {
-          $type: 'com.example.newModel',
-          name: 'This is a new model',
-        },
-      },
-    });
-    await settled();
-
-    assert.ok(
-      this.debugAdapter.addedType.calledWithExactly('com.example.new-model'),
-      'addedType is called when new model types are pushed into the store'
-    );
-  });
-
   test('addedType wraps model and notifies Ember Inspector of new types', async function(assert) {
```

Comment:
I think I can remove this test because the test for `watchModelTypes` in `interop-debug-adapter-test.js` handles this case in a better way but let me know if you have any concerns.

Reply:
yep i agree 👍